cidc_api/models/files/facets.py

```diff
@@ -200,6 +200,16 @@ def __init__(self, match_clauses: List[str], description: Optional[str] = None):
         "Images": FacetConfig(["/ihc/ihc_image."]),
         "Combined Markers": FacetConfig(["csv|ihc marker combined"]),
     },
+    "TCR": {
+        "Source": FacetConfig(
+            [
+                "/tcr/r1.fastq.gz",
```

Comment:
I don't think that's how facets will look like. There's a code that converts gcs uris into those, please check it out.